src/components/BlogPostComponent.vue

```diff
@@ -1,36 +1,14 @@
 <template>
-  <div class="card" style="width: 25rem;">
-    <div class="card-body">
-      <img v-bind:src="this.post.coverImageURL" width="100" height="100" />
-      <h5 class="card-title">{{ this.post.title }}</h5>
-      <p class="card-text">{{ this.subtitle }}</p>
-      <h5 class="author">by {{ this.post.User.firstName }} {{ this.post.User.lastName }}</h5>
-      <a href="#" class="btn btn-primary">Go somewhere</a>
-    </div>
+  <div>
+    This will eventually be used to display one single blog post but for now it is not needed.
```

Comment:
This is a blog post component that we aren't using yet. I left it in here but changed the content of it. We may actually be able to delete this altogether